### DISCUSSION TITLE
Help for possible incomplete patch process

### DISCUSSION THREAD

**Discussion by archercj** [Mr-MIBonk/M.I.B._More-Incredible-Bash#215] [General]
I updated the FW and it all went fine. Everything rebooted fine, radio playing etc etc. The only thing that wouldnt work were the maps - wouldnt load them. So thinking I need to patch the new firmware with the FEC patch I proceeded with the MIB process.

But now the unit boots up but nothing is operational - maps say initialising to 11%, radio wont play etc etc.

[MU1329-JTB-07415.11.1611900687.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11509251/MU1329-JTB-07415.11.1611900687.log)

I looked at the attached log file and noticed the difference to my previous update is that the following is missing from the log (this is from my previously successful FEC update): 

-----------
timestamp: 1970_01_01_00_06_45

MMX-showimage---->
Reboot Unit...

I dont seem to be able to get into the engineering menu either (although I can system reboot with the correct keys).

Can you give any advice please?

**Comment by harman-f** [comment]
Not sure what you aks for.
You started a flash process, but no access to GEM?

Log shows a successful patch, but no Fec was added, which should have been done automatically as well.
Or did you not use AIO patch?

**Reply by harman-f** [reply]
Also get latest M.I.B version, the 3.6.0 you use is "old" a few days at least

**Reply by archercj** [reply]
Put the new MIB on. System noots to a logo screen but select any function and you get things like initilaising maps and sticking at 11%

And the engineer menu wont load up.

I think it has failed to complete the patch and as such the system wont fully load

**Comment by harman-f** [comment]
Log stops before flash is started. 
normally log should show all flash steps. 

Do you have D-link?

**Reply by archercj** [reply]
What is a D-Link. I'm in computing and its a router manufacturer

**Reply by harman-f** [reply]
https://mibwiki.one/doc/d-link-dub-100-ver-d1-0x2001-0x1a02-pXHeQsqbwy

**Reply by archercj** [reply]
Ahh right telnet onto it then
Do you think a not completed FEC patch process from MIB is fixable?

**Reply by harman-f** [reply]
should be fixable, have to undersatnd what is going on. LOG file does not show a clear cause

**Reply by archercj** [reply]
My thoughts are that the patch process has not completed. As such the MMI FW has an issue during booting. The MMI logo shows up but there is very limited button response. You do get 'initialising maps' but this process stops at 11%. You also get 'No radio stations present', No multimedia found' etc but you cant access the settings menu.

It also appears you cannot access the red engineer menu which I think is key. If I could I am sure I could run the GEM/MIB process again.

**Reply by archercj** [reply]
If it is going to be a D-Link connection to copy the backup back on to the system are there any step by step instructions available how to do this please?
